src/apps/tabs/assets/arrow_up.svg

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg width="15px" height="14px" viewBox="0 0 15 14" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:sketch="http://www.bohemiancoding.com/sketch/ns">
+    <!-- Generator: Sketch 3.2.2 (9983) - http://www.bohemiancoding.com/sketch -->
+    <title>Rectangle 24</title>
+    <desc>Created with Sketch.</desc>
+    <defs></defs>
```

Comment:
I usually pass the sketch generated svgs via a script which cleans all junk they have like `title`, `desk`, `defs`, ` xmlns:sketch`, ` sketch:[a-z]*`, `id`. 

Unfortunately I had forgotten to commit this script in my config files and removed it by accident so time ago,